CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,7 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Network/NBD] Add the possibility to add and change the NBD connection associated to a Network (PR [#6646](https://github.com/vatesfr/xen-orchestra/pull/6646))
```

Comment:
Put it at the end of the list, we keep the list sorted chronologically.

Reply:
understood!